tests/bluetooth/bt_crypto/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_ENABLE_CRYPTO
```

Comment:
Why is this Kconfig option needed?

Reply:
Because `BT_CRYPTO` cannot be selected by the user, it can't be added in `prj.conf`.

Reply:
but then you can just do:
```
config BT_CRYPTO
	default y
```

no need for a new Kconfig.